site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+
+## API
+
+### Component properties
+
+Common texts, groups and cookie data can be found in <Link href="">HDS-provided content file</Link>. The file contains all user interface strings, language options, descriptions for common cookie groups, and details about common Helsinki cookies. By default, this content file is used. You can also override parts of it and add your project-specific groups and cookies. 
+
+#### ContentSource
+The `contentSource` property lists all content, language options, and callbacks for the component. The given source is merged with the default `content.json` to create a content-object.
+
+| Property                                                                                                                         | Required | Description                                                                                                                                                                                                                                                                                                            |
+| -------------------------------------------------------------------------------------------------------------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `siteName`                                                                                                                       | **Yes**  | Cookie consent main heading.                                                                                                                                                                                                                                                                                           |
+| `currentLanguage`                                                                                                                | **Yes**  | Currently selected language. Used for picking translations from `content.json`. Is also set to `content.language.current`. Supported values are `fi`, `sv`, and `en`.                                                                                                                                                   |
+| `requiredCookies`                                                                                                                | No       | Required cookies can have partial data or all data described in the table below. Missing texts are auto-filled. See examples below this table.                                                                                                                                                                         |
```

Comment:
Would it be a good idea to add link to the table (to the nearest header) where the content of `requiredCookies` is described in more detail? It could help understanding the complex structure of the content source.

Reply:
Improved